constants/appLinks.ts

```diff
@@ -50,6 +50,7 @@ const TEXTBOOK_BETA_START_LEARNING: GeneralLink = {
 }
 
 const textbookBetaIntroductionCourseStartLearningUrl = 'https://learn.qiskit.org/course/introduction'
+const textbookBetaQuantumMachineLearningCourseStartLearningUrl = 'https://learn.qiskit.org/course/machine-learning'
```

Comment:
I realize you're following previous naming conventions, but I wonder if it'd be easier to digest if we rename this to something like

`textbookBetaQMLStartLearningUrl`

To be clear, this is not your fault and not a blocker... just my instinct is that these variables are getting a little lengthy, and are now even longer than the string they're representing.